src/PSDMatrices.jl

```diff
@@ -12,7 +12,7 @@ end
 
 # Base overloads
 
-size(M::PSDMatrix) = (size(M.R, 1), size(M.R, 1))
+size(M::PSDMatrix) = (size(M.R, 2), size(M.R, 2))
```

Comment:
That was a leftover error from #13, which could only now be detected because for square matrix square roots that did not matter.